runtime/frequency-rococo/src/lib.rs

```diff
@@ -287,7 +289,7 @@ impl pallet_timestamp::Config for Runtime {
 	type Moment = u64;
 	type OnTimestampSet = ();
 	type MinimumPeriod = MinimumPeriod;
-	type WeightInfo = ();
+	type WeightInfo = weights::pallet_timestamp::SubstrateWeight<Runtime>;
```

Comment:
I've see a few chains with this set as `()`. Are we sure about this one?

Reply:
Yes.  In Moonbeam (for timestamp) it's explicitly `pallet_timestamp::weights::SubstrateWeight<Runtime>`.
In Acala they are using defaults `()`.
However, Kilt has run benchmarks , including timestamp and session, and uses these weights in their runtime configs, such as [peregrine](https://github.com/KILTprotocol/kilt-node/blob/develop/runtimes/peregrine/src/lib.rs).
 
Parity says "Benchmarking the Pallets is always necessary since the results depend on your runtime configuration. Each variable that you change in the runtime config can lead to different results for that Pallet."

This is why we have been running those benchmarks for the others, such as orml and democracy, and this PR simply adds the ones that we hadn't run benchmarks for yet, generally because they were added later. Moonbeam and Acala simply have not done this yet.

This [Question/Answer on Substrate StackExchange may help](https://substrate.stackexchange.com/questions/3229/benchmark-frame-pallets) for future reference.

Reply:
Sounds good. Thanks for checking in and the great explanation.